src/interfaces/issue.ts

```diff
@@ -6,10 +6,11 @@ import {components} from '@octokit/openapi-types';
 export interface IIssue {
   title: string;
   number: number;
-  created_at: IsoDateString;
-  updated_at: IsoDateString;
+  createdAt: IsoDateString;
+  updatedAt: IsoDateString;
```

Comment:
As soon as the rest API is dropped it might have sense to make these properties `Date` to avoid the converting to dates here and there throught the code